ts/smelter/src/components/Mp4.ts

```diff
@@ -20,26 +20,32 @@ export type Mp4Props = Omit<ComponentBaseProps, 'children'> & {
 
   /**
    *  Url, path to the mp4 file or `Blob` of the mp4 file. File path refers to the filesystem where Smelter server is deployed.
+   *  `Blob` is only supported on `@swmansion/smelter-web-wasm`.
    */
   source: string | Blob;
 };
 
 function Mp4(props: Mp4Props) {
-  const { muted, volume, ...otherProps } = props;
+  const { muted, volume, source, ...otherProps } = props;
   const ctx = useContext(SmelterContext);
   const [inputId, setInputId] = useState(0);
 
   useEffect(() => {
     const newInputId = newInternalStreamId();
     setInputId(newInputId);
 
-    let source: Pick<RegisterMp4Input, 'url' | 'serverPath' | 'blob'>;
-    if (props.source instanceof Blob) {
-      source = { blob: props.source };
-    } else if (props.source.startsWith('http://') || props.source.startsWith('https://')) {
-      source = { url: props.source };
+    let sourceObject: Pick<RegisterMp4Input, 'url' | 'serverPath' | 'blob'>;
+    if (source instanceof Blob) {
+      sourceObject = { blob: source };
+    } else if (source.startsWith('http://') || source.startsWith('https://')) {
+      sourceObject = { url: source };
     } else {
-      source = { serverPath: props.source };
+      sourceObject = { serverPath: source };
+    }
+
+    // If blob and run on Node.js
+    if (sourceObject.blob && typeof window === 'undefined') {
+      throw new Error('Blob is not supported on Node.js');
```

Comment:
Instead of this, we could use different `Mp4Props` depending on whether the type is imported from wasm or node